tools/machine-tools/setup_machines/README.md

```diff
@@ -0,0 +1,268 @@
+# Introduction
+
+In this development environment, deploying kubernetes involves two steps.
+
+- Create machines with required binaries, such as kubelete, kubeadm and docker
+- Create a kubernetes cluster using tools such as kubeadm
+
+The scripts under `./providers/kinder/` provide the solution to both of
+the above tasks as follows:
+
+**Creating Machines**: Using the scripts under `/providers/kinder/`, it
```

Comment:
There is "providers/kinder" nowhere to be found in Metal3/Nordix, but this document. Is this content relevant still?